CONTRIBUTING.md

```diff
@@ -0,0 +1,9 @@
+How to contribute to disq-bio
+=============================
+
+Thank you in advance for contributing to disq-bio!
+
+[todo]
```

Comment:
I realized that you added to every PR the `[DISQ-{{issue_number}}]` and that should be documented somewhere, in addition to add the `Fixes/Closes #{{issue_number}}` for cross-reference. On the other hand, what's about PRs without associated issue?

Maybe a good starting point to add something to this file?